.idea/.generators

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
aha .idea/ is a rubymine thing not a homepointr thing so could you create a global gitignore and put a couple things there? 

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

then go to your home directory and open your global gitignore and add what I send you on slack